docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -380,14 +380,16 @@ There is no limit on field descriptions for either key or message.
 Confluent table description supplier
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Confluent table description supplier uses the `Confluent Schema Registry
-<https://docs.confluent.io/1.0/schema-registry/docs/intro.html>`_ to discover
-table definitions. Set ``kafka.table-description-supplier`` to ``CONFLUENT`` to
-use it and configure the additional properties in the following table.
-
-The benefits of using the schema registry is that new tables can be defined
-without a cluster restart, schema updates are detected and there is no need to
-define tables manually.
+The Kafka connector only uses the `Confluent Schema Registry
+<https://docs.confluent.io/1.0/schema-registry/docs/intro.html>`_ to
+discover table definitions. The benefits of using the schema registry are:
```

Comment:
If the intent is to indicate it only works with the Confluent Schema Registry then would this be better? Current wording seems confusing and doesn't convey what we want to convey.

```suggestion
The Confluent table description supplier uses the `Confluent Schema Registry
<https://docs.confluent.io/1.0/schema-registry/docs/intro.html>`_ to discover
table definitions. Set ``kafka.table-description-supplier`` to ``CONFLUENT`` to
use it and configure the additional properties in the following table.
This table description supplier is only tested to work with the Confluent Schema Registry [but may or may not work with other compatible schema registries].
```

Feel free to drop the part in `[]` depending on what we want to convey.
```